### PR TITLE
Change/add content for data space details

### PR DESCRIPTION
**Pull request by jjeroch**
## Description

Dataspace Policy Management Content

## Why

Description and important links added to the de/en data policy management introduction page which got recently implemented.

## Issue

n/a

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by jjeroch** [comment]
@manojava-gk please do me a favor and add for the use case section as well as the roles the catena-x logo (default image) instead of the "Access/Usage" image. Thanks.
You can do the change in the same pr if you like.

Additionally please review this pr.

File: public/assets/content/de/dataspace.json [hunk 1]
```diff
@@ -82,16 +82,16 @@
       },
       {
         "title":"Use Cases",
-        "description":"This text need to get added still<1></1><1></1>>> BPDM Use case details (coming soon)",
-        "imagePath":"/images/frame/AccessVsUsage.png",
+        "description":"Unterstützte UseCase-Anmeldeinformationen: <1></1><1></1>>> Attribute - >>> CredentialSubjectItem <1></1><1></1>>> FrameworkAgreement.pcf - PcfCredential <1></1><1></1>>> FrameworkAgreement.sustainability - SustainabilityCredential <1></1><1></1>>> FrameworkAgreement.quality - QualityCredential <1></1><1></1>>> FrameworkAgreement.traceability - TraceabilityCredential <1></1><1></1>>> FrameworkAgreement.behavioraltwin - BehaviorTwinCredential <1></1><1></1>",
```
Comment (manojava-gk) [inline public/assets/content/de/dataspace.json:85]: Screenshot 2023-07-18 at 12 31 02 PM

Comment (manojava-gk) [reply public/assets/content/de/dataspace.json:85]: This is how it displays

File: public/assets/content/en/dataspace.json [hunk 2]
```diff
@@ -82,16 +82,16 @@
       },
       {
         "title":"Use Cases",
-        "description":"This text need to get added still<1></1><1></1>>> BPDM Use case details (coming soon)",
-        "imagePath":"/images/frame/AccessVsUsage.png",
+        "description":"Supported UseCase credentials: <1></1><1></1>>> Attribute - >>> CredentialSubjectItem <1></1><1></1>>> FrameworkAgreement.pcf - PcfCredential <1></1><1></1>>> FrameworkAgreement.sustainability - SustainabilityCredential <1></1><1></1>>> FrameworkAgreement.quality - QualityCredential <1></1><1></1>>> FrameworkAgreement.traceability - TraceabilityCredential <1></1><1></1>>> FrameworkAgreement.behavioraltwin - BehaviorTwinCredential <1></1><1></1>",
```
Comment (manojava-gk) [inline public/assets/content/en/dataspace.json:85]: same as above

**Review by manojava-gk — COMMENTED** [review]
small finding/clarification

File: public/assets/content/de/dataspace.json (hunk 1 above)
Comment (jjeroch) [reply public/assets/content/de/dataspace.json:85]: yes, thats how it is expected

File: public/assets/content/en/dataspace.json (hunk 2 above)
Comment (jjeroch) [reply public/assets/content/en/dataspace.json:85]: as expected

**Review by manojava-gk — APPROVED** [review]
lftm

**Comment by manojava-gk** [comment]
@jjeroch kindly review the new contents once

**Review by oyo — APPROVED** [review]

**Pull request merged**